exonum-java-binding-common/src/test/java/com/exonum/binding/common/proofs/list/ListProofUtils.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2018 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.common.proofs.list;
+
+import static com.exonum.binding.common.hash.Funnels.hashCodeFunnel;
+
+import com.exonum.binding.common.hash.HashCode;
+import com.exonum.binding.common.hash.Hashing;
+import com.exonum.binding.common.hash.PrimitiveSink;
+import com.exonum.binding.common.serialization.StandardSerializers;
+import java.nio.charset.StandardCharsets;
+import java.util.Optional;
+
+/**
+ * Various utilities for testing ListProof verification {@link ListProofNode}s.
+ */
+final class ListProofUtils {
+
+  private ListProofUtils() {
+  }
+
+  /**
+   * Creates ListProof element node.
+   */
+  static ListProofElement leafOf(String element) {
+    byte[] dbElement = bytesOf(element);
+    return new ListProofElement(dbElement);
+  }
+
+  /**
+   * Returns string bytes. Standard string serialized is used.
```

Comment:
Maybe you meant `string serializer`?

Reply:
No, bytesOf() returns bytes.

Reply:
I meant "Returns string bytes. Standard string serialize~~d~~r is used"